src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormMain.tsx

```diff
@@ -0,0 +1,207 @@
+import {
+  Box,
+  Button,
+  Clickable,
+  DROP_SHADOW,
+  Flex,
+  Spacer,
+  Text,
+  useToasts,
+} from "@artsy/palette"
+import { AppContainer } from "Apps/Components/AppContainer"
+import { ConfirmationModalBack } from "Apps/MyCollection/Routes/EditArtwork/Components/ConfirmationModalBack"
+import { ConfirmationModalDelete } from "Apps/MyCollection/Routes/EditArtwork/Components/ConfirmationModalDelete"
+import { useMyCollectionArtworkFormContext } from "Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormContext"
+import { MyCollectionArtworkFormDetails } from "Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormDetails"
+import { MyCollectionArtworkFormHeader } from "Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormHeader"
+import { MyCollectionArtworkFormImages } from "Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormImages"
+import { useDeleteArtwork } from "Apps/MyCollection/Routes/EditArtwork/Mutations/useDeleteArtwork"
+import { ArtworkModel } from "Apps/MyCollection/Routes/EditArtwork/Utils/artworkModel"
+import { useMyCollectionTracking } from "Apps/MyCollection/Routes/Hooks/useMyCollectionTracking"
+import { Form, useFormikContext } from "formik"
+import { useState } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useRouter } from "System/Router/useRouter"
+import createLogger from "Utils/logger"
+import { Media } from "Utils/Responsive"
+import { MyCollectionArtworkFormMain_artwork$data } from "__generated__/MyCollectionArtworkFormMain_artwork.graphql"
+
+const logger = createLogger("MyCollectionArtworkForm.tsx")
+
+export interface MyCollectionArtworkFormMainProps {
+  artwork?: MyCollectionArtworkFormMain_artwork$data
+}
+
+export const MyCollectionArtworkFormMain: React.FC<MyCollectionArtworkFormMainProps> = ({
```

Comment:
Moving most from the old `MyCollectionArtworkForm` component to `MyCollectionArtworkFormMain`.